docs/baseline_results/cifar10_witches_brew_results.md

```diff
@@ -1,3 +1,146 @@
 # Cifar10 Witches' Brew Baseline Evaluation
 
-Coming soon
\ No newline at end of file
+Results obtained using Armory 0.15.X (July 2022)
+
+Note: the baseline CIFAR resnet was updated in Armory 0.16.1.
```

Comment:
if the model was updated in 0.16.1 but the results are from 0.15, does this mean the results need to be re-computed?

Reply:
Eventually I suppose that would be nice; it's certainly not urgent since witches' brew is not going to be a poisoning scenario we continue to use.  I think that labelling each result page with the armory version gives us some flexibility in when we update the results